build/debian-hyperkube-base/Makefile

```diff
@@ -23,7 +23,7 @@ TAG=0.10
 ARCH?=amd64
 CACHEBUST?=1
 
-BASEIMAGE=k8s.gcr.io/debian-base-$(ARCH):0.3
```

Comment:
shouldn't `PROJECT_ID` have a default? This also doesn't allow pushing to the previous path at all, perhaps just allow overriding `BASEIMAGE` entirely or have a `BASEREGISTRY` (or both?)

```
BASEREGISTRY?=k8s.gcr.io
BASEIMAGE?=${BASE_REGISTRY}/debian-base-$(ARCH):0.3
```

Reply:
Er yeah, I just made the current config overridable in my example, since that seemed like the relevant part of this PR. We should also really make it possible to push these images to other locations.

Reply:
debian-base and hyperkube are not e2e test images, so they should continue to live in gcr.io/google-containers (i.e. k8s.gcr.io).